src/WebJobs.Script.WebHost/Diagnostics/Extensions/ScriptHostServiceLoggerExtension.cs

```diff
@@ -330,17 +330,33 @@ public static void ScriptHostServiceRestartCanceledByRuntime(this ILogger logger
             _scriptHostServiceRestartCanceledByRuntime(logger, null);
         }
 
-        public static void LogHostInitializationSettings(this ILogger logger, string functionWorkerRuntime, string functionExtensionVersion, string siteExtensionDirectory, bool inStandbyMode)
+        public static void LogHostInitializationSettings(this ILogger logger, string originalFunctionWorkerRuntime, string functionWorkerRuntime,
+            string originalFunctionWorkerRuntimeVersion, string functionsWorkerRuntimeVersion, string functionExtensionVersion, string currentDirectory,
+            bool inStandbyMode, bool hasBeenSpecialized, bool usePlaceholderDotNetIsolated, string websiteSku, string featureFlags,
+            IDictionary<string, string> hostingConfig)
         {
-            var hostInitializationSettings = $@"{{
-    ""HostInitializationSettings"": {{
-        ""functionsWorkerRuntime"": ""{functionWorkerRuntime}"",
-        ""functionsExtensionVersion"": ""{functionExtensionVersion}"",
-        ""siteExtensionDirectory"": ""{siteExtensionDirectory}"",
-        ""inStandbyMode"": {inStandbyMode}
-    }}
-}}";
-            _logHostInitializationSettings(logger, hostInitializationSettings, null);
+            // This is a dump of values for telemetry right now, but eventually we will refactor this
+            // into a proper Options object
+            var initializationSettings = new
+            {
+                OriginalFunctionWorkerRuntime = originalFunctionWorkerRuntime,
```

Comment:
`Original` here means  `Placeholder`. Correct?

Reply:
Correct. But not all hosts are specialized, and I didn't want to make this too crazily complicated and only write some values when specialize and leave ones off when not specialized, etc. So, I decided this would hopefully be clear enough for us to pick apart the values we wanted for reports. In some cases of pre-specialization and non-specialization, these values will always be the same. But I think that's okay.